Published versions of ERTP and Zoe seem to have error

**Describe the bug**

Trying to import Zoe from npm gives the following `Error: Cannot find module '@agoric/ertp/exported'`
<img width="670" alt="Screen Shot 2020-09-24 at 6 06 49 PM" src="https://user-images.githubusercontent.com/2441069/94215290-0b35a700-fe91-11ea-9b07-096b64de100c.png">

I believe this is caused by ERTP not including exported.js in its `files` (the list of files that gets published to npm):

<img width="391" alt="Screen Shot 2020-09-24 at 6 14 53 PM" src="https://user-images.githubusercontent.com/2441069/94215610-0fae8f80-fe92-11ea-8544-d7ce492f02d7.png">

**To Reproduce**
Steps to reproduce the behavior:
1. Create a new directory.
2. `npm init`
3.  `yarn add @agoric/zoe`
4. `yarn add esm`
5. Create a file `index.js` with the following:
```js
import { makeZoe } from '@agoric/zoe';
makeZoe();
```
6. Run the code: `node -r esm index.js`
7. You will get the error message `Cannot find module '@agoric/ertp/exported'`

**Expected behavior**
Using ERTP and Zoe from npm shouldn't have errors, although we have not been testing for that specifically.

**Platform Environment**
macOs, node v14.11.0

--- a/packages/ERTP/package.json
+++ b/packages/ERTP/package.json
@@ -58,7 +58,8 @@
   },
   "files": [
     "src",
-    "NEWS.md"
+    "NEWS.md",
+    "exported.js"
   ],
   "ava": {
     "files": [

